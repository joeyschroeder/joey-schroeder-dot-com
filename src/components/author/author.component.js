import React from 'react';
import styles from './author.style.scss';

export function Author() {
  return <h1 className={styles.root}>Joey Schroeder</h1>;
}
