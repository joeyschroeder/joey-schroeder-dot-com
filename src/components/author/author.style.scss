@import '../../styles/config/variables';
@import '../../styles/mixins/font-size';
@import '../../styles/mixins/line-height';

@keyframes authorMove {
  0% {
    opacity: 0;
    transform: translateY(-300%);
  }

  100% {
    opacity: 1;
    transform: translateY(0%);
  }
}

.root {
  @include font-size(2.4);
  @include line-height(2.4);

  animation-delay: $duration-base * 2;
  animation-duration: $duration-base * 4;
  animation-fill-mode: both;
  animation-iteration-count: 1;
  animation-name: authorMove;
  animation-timing-function: ease;
  font-weight: 200;
  margin: 0;
  text-transform: uppercase;
}
