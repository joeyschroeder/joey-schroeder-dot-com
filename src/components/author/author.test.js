import React from 'react';
import { create } from 'react-test-renderer';
import { Author } from './author.component';

describe('<Author />', () => {
  it('should render correctly', () => {
    const component = create(<Author />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
