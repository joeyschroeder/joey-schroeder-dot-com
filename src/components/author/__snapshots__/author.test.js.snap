// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Author /> should render correctly 1`] = `
<h1
  className="root"
>
  Joey Schroeder
</h1>
`;
